packages/documentation/src/content/docs/integration/requirements/idp.mdx

```diff
@@ -12,179 +12,130 @@ import {
-| Invalid  | The grant was not in a state where it could be accepted or rejected (e.g., grant was already approved)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          | `?result=grant_invalid`                                                                                                          |
+| Response | Description | Example | 
+| -------- | ----------- | ------- |
+| Rejected | The interaction was rejected by the end-user | `?result=grant_rejected` |
+| Invalid  | The grant was not in a state where it could be accepted or rejected (e.g., grant was already approved) | `?result=grant_invalid` |
 | Success  | The grant was successful with the following returned in the response:<br /><ul><li>A hash representing the SHA-256 hash of values provided by the client in the <LinkOut href="https://openpayments.dev/apis/auth-server/operations/post-request/">grant initialization request</LinkOut> (`interact.finish.nonce`), and the values in the response returned from your authorization server (`interact.finish`).</li><li>The `interact_ref` that identifies the interaction on your authorization server alongside the hash</li><li>The URI of the grant initialization request (e.g., `https://www.auth-server.com`)</li></ul> | `hash=p28jsq0Y2KK3WS__a42tavNC64ldGTBroywsWxT4md_jZQ1R\HZT8BOWYHcLmObM7XHPAdJzTZMtKBsaraJ64A &interact_ref=4IFWWIKYBC2PQ6U56NL1` |
 
 </div>
 
-#### Continue grant
+When successful, the SHA-256 hash of the interaction is sent in the response to the client, along with an `interact_ref` that identifies the interaction on the authorization server and the URI of the grant initialization request. The client must verify the hash before it will request to continue the grant
 
-Called by the client to request a grant from your authorization server if the interaction was successful (accepted). Your authorization server responds with an <LinkOut href="https://openpayments.dev/apis/auth-server/operations/post-continue/">access token</LinkOut>.
+#### Continue grant
 
-### x-idp-secret header
+The client requests a grant from your authorization server for an accepted interaction. Your authorization server responds with an <LinkOut href="https://openpayments.dev/apis/auth-server/operations/post-continue/">access token</LinkOut>.
```

Comment:
Technically speaking, there is a single grant (ie the grant was initially requested, but needed a interaction to complete before "continue" was called, and the grant access token be issued to the client).

Reply:
This section no longer makes mention of grants or access tokens. It now focuses on the header itself and its purpose (securing communications).